manager/orchestrator/task.go

```diff
@@ -119,6 +119,10 @@ func nodeMatches(s *api.Service, n *api.Node) bool {
 		return false
 	}
 
+	if s.Spec.Task.Placement == nil {
+		return false
```

Comment:
IMHO this should be true: no placement constraint means any node is suited to host that task.

Reply:
You are probably right